Algebraic Topology/algebraic-topology.tex

```diff
@@ -224,18 +225,18 @@ \section{Homotopy}
 \end{thm}
 \begin{thm}
   If $X$ is path connected, then for any $x_a, x_b \in X$,
-  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$
-  notation If $X$ is path connected, we can denote the fundamental
+  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$ %
+  Notation: If $X$ is path connected, we can denote the fundamental
   group with $\pi_1(X)$ or $\pi_1 X$.
 \end{thm}
 \begin{example}
-  pretty picture
+  pretty picture %
 
   This isomorphism is not canonical
   
 
   Note that $\pi_1 X$ is only well defined up to inner automorphism.
-  ((Chris, what does this mean?))
+  ((Chris, what does this mean?)) %
```

Comment:
George you are correct as far as I understand, you are able to translate pi_1 around from base point to base point via conjugation.